requirements/dev.txt

```diff
@@ -309,7 +309,7 @@ rfc3986==2.0.0
     # via
     #   -r requirements/quality.txt
     #   twine
-rich==12.5.1
+rich==12.6.0
```

Comment:
No breaking change in the new version: https://github.com/Textualize/rich/blob/master/CHANGELOG.md#1260---2022-10-02